frontend/src/api/k8s/groups.ts

```diff
@@ -0,0 +1,14 @@
+import { k8sListResource } from '@openshift/dynamic-plugin-sdk-utils';
```

Comment:
Both groups and users are for typeahead, instead of debouncing (which is impossible to filter by regex) I decided to include that in the project context, we can scrap the typeahead if we don't find it useful or doesn't fit the performance needed.

Reply:
If you can't filter users, we can't use it. You should not fetch every user or the "first set" -- as it won't be useful to the user once they start typing. I'll have to look into the code (just viewing this at a high level right now)

Reply:
Yeah, there is no regex operation with field selectors, [documentation here](https://kubernetes.io/docs/concepts/overview/working-with-objects/field-selectors/) so we might just wanna have this in his own context or remove it. cc @kywalker-rh